README.md

```diff
@@ -109,6 +109,7 @@ Project can be deployed by using the windows ,linux servers.
 
 - [ ] **More react features**
 - [ ] **API implemetation**
+      ** More componenets **
```

Comment:
When I started your app with `npm start`, I encountered this error 👀
![image](https://github.com/samuelkinyanjui/bookstore2/assets/78251927/e77db84a-7701-4dd1-868b-f5040c94f0db)

Which means `index.html` has been removed from the /public directory. I could see that the entire /public directory has been removed from your app, you can create another app using create-react-app, then copy and paste the public folder. It should be in `<your project root>/public/index.html` for everything to work as expected

Create React App folder structure
```
├── README.md
├── node_modules
├── package.json
├── .gitignore
├── public
│   ├── favicon.ico
│   ├── index.html
│   ├── logo192.png
│   ├── logo512.png
│   ├── manifest.json
│   └── robots.txt
└── src
    ├── App.css
    ├── App.js
    ├── App.test.js
    ├── index.css
    ├── index.js
    ├── logo.svg
    ├── serviceWorker.js
    └── setupTests.js
```